scripts/tests/test_question_answering.py

```diff
@@ -21,17 +21,15 @@
 
 from mxnet.gluon.data import DataLoader, SimpleDataset
 
-from gluonnlp.data import SQuAD
 from ..question_answering.data_processing import SQuADTransform, VocabProvider
 
 question_max_length = 30
 context_max_length = 256
 
 
 @pytest.mark.remote_required
-def test_transform_to_nd_array():
-    dataset = SQuAD(segment='dev', version='1.1', root='tests/data/squad')
-    vocab_provider = VocabProvider(dataset)
+def test_transform_to_nd_array(squad_dev_and_vocab_provider):
```

Comment:
why is `squad_dev_and_vocab_provider` an argument?

Reply:
it loads from the fixture